scripts/gha/integration_testing/xcodebuild.py

```diff
@@ -79,14 +80,24 @@ def get_args_for_build(path, scheme, output_dir, ios_sdk, configuration):
   return args
 
 
-def _get_ios_env_from_target(ios_sdk):
+def _get_ios_env_from_target(ios_sdk, target_os):
   """Return a value for the -sdk flag based on the target (device/simulator)."""
-  if ios_sdk == "device":
-    return "iphoneos"
-  elif ios_sdk == "simulator":
-    return "iphonesimulator"
+  if target_os == "iOS":
+    if ios_sdk == "device":
+      return "iphoneos"
+    elif ios_sdk == "simulator":
+      return "iphonesimulator"
+    else:
+      raise ValueError("Unrecognized iOS ios_sdk paramter: %s" % ios_sdk)
+  elif target_os == "tvOS":
+    if ios_sdk == "device":
+      return "appletvos"
+    elif ios_sdk == "simulator":
+      return "appletvsimulator"
+    else:
+      raise ValueError("Unrecognized tvOS ios_sdk parameter: %s" % sdk)
   else:
-    raise ValueError("Unrecognized ios_sdk: %s" % ios_sdk)
+    raise ValueError("Unrecognized target_os %s" % target_os)
```

Comment:
nit: probably also worth to log what's the ios_sdk here.

Reply:
Good idea. Done!